### PR TITLE
Add a more detailed introduction for model compression

### PR DESCRIPTION
**Pull request by denghuilu**

### CODE REVIEW

File: deepmd/entrypoints/compress.py [hunk 1]
```diff
@@ -34,9 +34,9 @@ def compress(
     """Compress model.
 
     The table is composed of fifth-order polynomial coefficients and is assembled from
-    two sub-tables. The first table takes the stride(parameter) as it's uniform stride,
-    while the second table takes 10 * stride as it's uniform stride. The range of the
-    first table is automatically detected by deepmd-kit, while the second table ranges
+    two sub-tables. The first table takes the step parameter as the domain's uniform step size,
+    while the second table takes 10 * step as it's uniform step size. The range of the
+    first table is automatically detected by the code, while the second table ranges
     from the first table's upper boundary(upper) to the extrapolate(parameter) * upper.
```
Comment (tuoping) [inline deepmd/entrypoints/compress.py:40]: I don't understand what is "upper", it is not in the parameter list.

File: doc/getting-started.md [hunk 2]
```diff
@@ -244,23 +244,44 @@ positional arguments:
...
+  -m {master,collect,workers}, --mpi-log {master,collect,workers}
+                        Set the manner of logging when running with MPI.
+                        'master' logs only on main process, 'collect'
+                        broadcasts logs from workers to master and 'workers'
+                        means each process will output its own log (default:
+                        master)
   -i INPUT, --input INPUT
                         The original frozen model, which will be compressed by
-                        the deepmd-kit
+                        the deepmd-kit (default: frozen_model.pb)
```
Comment (tuoping) [inline doc/getting-started.md:262]:
```suggestion
 (default: frozen_model.pb)
```

File: doc/getting-started.md [hunk 3]
```diff
@@ -244,23 +244,44 @@ positional arguments:
...
+                        specified, the logs will only be output to console
+                        (default: None)
+  -m {master,collect,workers}, --mpi-log {master,collect,workers}
+                        Set the manner of logging when running with MPI.
+                        'master' logs only on main process, 'collect'
+                        broadcasts logs from workers to master and 'workers'
+                        means each process will output its own log (default:
+                        master)
   -i INPUT, --input INPUT
                         The original frozen model, which will be compressed by
```
Comment (tuoping) [inline doc/getting-started.md:261]:
```suggestion
                        The original frozen model, which will be compressed
```

File: doc/getting-started.md (hunk 2 above)
Comment (tuoping) [inline doc/getting-started.md:262]:
```suggestion
                        The original frozen model, which will be compressed. (default: frozen_model.pb)
```

Comment (tuoping) [inline doc/getting-started.md:262]:
```suggestion
which will be compressed (default: frozen_model.pb)
```

File: doc/getting-started.md [hunk 4]
```diff
@@ -244,23 +244,44 @@ positional arguments:
...
   -o OUTPUT, --output OUTPUT
-                        The compressed model
+                        The compressed model (default:
+                        frozen_model_compress.pb)
+  -s STEP, --step STEP  Model compression introduces two tables with different
+                        uniform step size to store the parameters of the
+                        fifth-order polynomials. The step parameter denotes
+                        the uniform step size of the first table and the
+                        second table will use 10 * step as it's uniform step
+                        size. Smaller step with higher accuracy and bigger
```
Comment (tuoping) [inline doc/getting-started.md:271]:
```suggestion
                        size. Smaller step means higher accuracy and bigger
```

File: doc/getting-started.md [hunk 5]
```diff
@@ -244,23 +244,44 @@ positional arguments:
...
-  -s STRIDE, --stride STRIDE
-                        The uniform stride of tabulation's first table, the
-                        second table will use 10 * stride as it's uniform
-                        stride
+                        The domain range of the first table is automatically
+                        detected by the code, while the second table ranges
+                        from the first table's upper boundary(upper) to the
+                        extrapolate(parameter) * upper. (default: 5)
   -f FREQUENCY, --frequency FREQUENCY
                         The frequency of tabulation overflow check(If the
```
Comment (tuoping) [inline doc/getting-started.md:279]:
```suggestion
                        The frequency of tabulation overflow check(Check to see whether the
```

File: deepmd/entrypoints/compress.py (hunk 1 above)
Comment (denghuilu) [reply deepmd/entrypoints/compress.py:40]: changed in the description.

**Review by tuoping — APPROVED** [review]

**Pull request merged**